### DISCUSSION TITLE
using Authelia for HTTP sites

### DISCUSSION THREAD

**Discussion by ghost** [authelia/authelia#5727] [Q&A]
Hi,
I have a HTTP site and want to use Authelia as SSO. I cannot redirect the requests to this site to HTTPS and it must be opened using HTTP. how do I set Authelia to be able to use it for HTTP site?
thanks.

**Comment by james-d-elliott** [comment]
We intentionally do not support HTTP as we set the cookie secure flag so the cookies cannot be sent over HTTP. Is there a specific reason you cannot do HTTPS?

**Comment by ghost** [comment]
Ok. no I was wrong. thank you.
